fluentd-image/v1.10/Dockerfile

```
@@ -1,5 +1,5 @@
 FROM alpine:3.11
-LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.10.4"
+LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.10.5"
```

```suggestion
LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.10.4"
```
This is still install fluentd version 1.10.4